src/main.js

```diff
@@ -9,7 +10,7 @@ const {
 } = require("./scripts/settings");
 const { addTray, refreshTray } = require("./scripts/tray");
 const { addMenu } = require("./scripts/menu");
-
+const clientId = '833403231153029210';
```

Comment:
Where did you get this from? Is this some kind of global key that is distributed from discord?

Reply:
> Where did you get this from? Is this some kind of global key that is distributed from discord?

It's an ID you get by making an application on Discord's site

Reply:
Clear! Just to avoid any confusion about who is using which access token I've made a new one in the name of tidal-hifi. Can you please use this one? :)

833617820704440341

Thanks!

Reply:
Alright! Make sure to also upload the logo under rich presence and then scroll down to assets and give me the name of the uploaded file so that it can actually show the tidal

Reply:
Sure. The name is "tidal-hifi-icon" for the black version and "tidal-hifi-icon-inverted" for the white version.

The black one is the default :)